pkg/helm/charts/tests/centralservices/testdata/helmtest/injected-cabundle-cm.test.yaml

```diff
@@ -16,15 +16,6 @@ tests:
   - name: "Injected CA bundle has a proper label"
     expect: |
       assertThat(.configmaps["injected-cabundle-stackrox-central-services"].metadata.labels["config.openshift.io/inject-trusted-cabundle"] == "true")
-- server:
-    visibleSchemas:
-    - openshift-3.11.0
-  set:
-    env.openshift: 3
-  tests:
-  - name: "No injected CA bundle on Openshift 3"
-    expect: |
-      assertThat(.configmaps["injected-cabundle-stackrox-central-services"] == null)
```

Comment:
Actually, why are we removing this? This has nothing to do with SCCs. Does it disturb somehow?

Reply:
This is the test that kept breaking after the Helm changes, so I removed it.

Reply:
Ouch, I would be careful here, maybe it was failing for a reason? Maybe try to wrap the head around that issue and try to understand why it is failing after the changes?

Reply:
Found the culprit in https://github.com/stackrox/stackrox/pull/8663/commits/1ab1f9556e816df583befefb3f8b7287d78f49c7. Looks like I had a search/replace going rogue. This is fixed now